airframe-rx-html/src/main/scala/wvlet/airframe/rx/html/RxElement.scala

```diff
@@ -34,10 +49,12 @@ abstract class RxElement(val modifiers: List[Seq[HtmlNode]] = List.empty) extend
     }
   }
 
-  def add(xs: HtmlNode*): RxElement =
+  def add(xs: HtmlNode*): RxElement = {
     new RxElement(xs :: modifiers) {
       override def render = self.render
```

Comment:
`beforeRender` and `beforeUnmount` also need to be overridden?

Reply:
Nice catch. I'll fix it.

Reply:
Actually, this part needed no fix if we change the logic of DOMRenderer to handle nested RxElements properly.